StateCodelab/start/src/main/java/com/codelabs/state/todo/TodoComponents.kt

```diff
@@ -129,17 +136,14 @@ private fun SelectableIconButton(
     } else {
         MaterialTheme.colors.onSurface.copy(alpha = 0.6f)
     }
-    Button(
+    TextButton(
         onClick = { onIconSelected() },
         shape = CircleShape,
-        backgroundColor = Color.Transparent,
-        border = null,
-        elevation = 0.dp,
+        colors = ButtonConstants.defaultTextButtonColors(),
```

Comment:
It's not needed, default value is already `ButtonConstants.defaultTextButtonColors()`